app/src/androidTest/java/com/github/bgabriel998/softwaredevproject/SettingsActivityTest.java

```diff
@@ -47,4 +64,12 @@ public void TestToolbarBackButton(){
             fail("TestToolbarBackButton failed");
         }
     }
+
```

Comment:
Head comment is missing
```suggestion
/* Test that offline mode button is working fine*/
```